nuxeo-ai-core/src/test/resources/OSGI-INF/threshold-config-test.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ai.threshold.test" version="1.0">
+
+  <extension target="org.nuxeo.ai.configuration.ThresholdComponent"
+             point="thresholdConfiguration" >
+    <thresholdConfiguration global="0.7">
+      <configurations>
+        <configuration type="File">
```

Comment:
Could this be the top level node, without the 2 above it?

Reply:
the idea that configuration can have multiple types/facets

Reply:
maybe if we take out the global threshold.